ecorise_dashboard.py

```diff
@@ -60,37 +61,78 @@
-'Workforce Development',
-'STEM']
-add_themes = ['Conservation (Wildlife/Habitat)', 'Green Building',  'Outdoor Learning']
-full_list = themes_list + add_themes
-
-def count_env_themes(programs, theme_list):
-    themes = programs[['Program','Environmental_Themes']]
-    expanded = themes['Environmental_Themes'].str.get_dummies(', ')
-    # Replace columns that aren't in list with 'Other'
-    expanded = expanded.rename(lambda x:  x if x in theme_list else 'Other', axis=1)
-    # Group 'Other' Columns together
-    expanded = expanded.groupby(expanded.columns, axis=1).sum()
-    # Merge the themes and expanded dataframes
-    themes = pd.concat([themes, expanded], axis=1)
-
-    # Get Count of Programs per theme
-    theme_count = pd.DataFrame(expanded.sum())
-    theme_count.reset_index(inplace=True)
-    theme_count.columns = ['Theme','Count']
-    theme_count['Percent'] = round(100 * theme_count['Count'] / len(programs),0)
-    theme_count = theme_count.sort_values(by=['Count'])
-    return theme_count
+# themes_list = ['Outdoor learning',
+# 'Waste',
+# 'Water',
+# 'Energy',
+# 'Green building',
+# 'Food Systems/Nutrition',
+# 'Transportation',
+# 'Air Quality',
+# 'Conservation (wildlife/habitat)',
+# 'Workforce Development',
+# 'STEM']
+# add_themes = ['Conservation (Wildlife/Habitat)', 'Green Building',  'Outdoor Learning']
+# full_list = themes_list + add_themes
+#
+# def count_env_themes(programs, theme_list):
+#     themes = programs[['Program','Environmental_Themes']]
+#     expanded = themes['Environmental_Themes'].str.get_dummies(', ')
+#     # Replace columns that aren't in list with 'Other'
+#     expanded = expanded.rename(lambda x:  x if x in theme_list else 'Other', axis=1)
+#     # Group 'Other' Columns together
+#     expanded = expanded.groupby(expanded.columns, axis=1).sum()
+#     # Merge the themes and expanded dataframes
+#     themes = pd.concat([themes, expanded], axis=1)
+#
+#     # Get Count of Programs per theme
+#     theme_count = pd.DataFrame(expanded.sum())
+#     theme_count.reset_index(inplace=True)
+#     theme_count.columns = ['Theme','Count']
+#     theme_count['Percent'] = round(100 * theme_count['Count'] / len(programs),0)
+#     theme_count = theme_count.astype({'Percent': int})
+#     theme_count = theme_count.sort_values(by=['Count'])
+#     return theme_count
+
+## Function to produce dataframe with: term, display_term
+def get_display_terms(filter_df, table, column):
+    filter_df = filter_df[filter_df['table_name'] == table]
+    filter_df = filter_df[filter_df['column_name'] == column]
+    filter_df = filter_df.loc[:, ['term','display_term']]
+    filter_df = filter_df.set_index('term')
+    return filter_df
+
+## Function to produce dataframe with: Environmental_Themes, Count, Percent
+def count_env_themes_for_programs(programs_df):
+    ## Get program names and environmental themes
+    program_themes_df = programs_df.loc[:, ['Program','Environmental_Themes']]

```

Comment:
Since you're selecting columns, this would be sufficient: 
```suggestion
    program_themes_df = programs_df[['Program','Environmental_Themes']]

```